tests/nativeapp/codegen/snowpark/__snapshots__/test_extension_function_utils.ambr

```diff
@@ -1,33 +1,97 @@
 # serializer version: 1
-# name: test_enrich_ex_fn
-  "'a/b/c.py','a/b/c','a/b/c.py','a/b/c.jar','a/b/c'"
+# name: test_deannotate_preserves_specified_annotations
+  '''
+  import snowflake.snowpark
+  
+  from typing import Optional
+  from snowflake.snowpark.session import Session
+  from snowflake.snowpark.functions import col, sum, sproc
+  
+  
+  @custom
+  #: @sproc(native_app_params={'schema': 'math', 'application_roles': ['app_public', 'app_admin']})
+  def sproc_sum(session: Session, first: int, second: int) -> int:
+      return first + second
+  
+  #: @udf(native_app_params={'schema': 'math', 'application_roles': ['app_public', 'app_admin']})
+  #: @module.annotation
```

Comment:
Why does this one get commented-out?

Reply:
+1

Reply:
Ahh ok, I missed that it was an allowlist (thought it was a denylist for some reason). Let me re-review with this context.